lib/al/Library/Action/ActorActionKeeper.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+namespace al {
+class ActorResource;
+class ActionAnimCtrl;
+class ActionFlagCtrl;
+class ActionEffectCtrl;
+class ActionSeCtrl;
+class ActionBgmCtrl;
+class ActionPadAndCameraCtrl;
+class ActionScreenEffectCtrl;
+class NerveActionCtrl;
+class LiveActor;
+
+class ActorActionKeeper {
+private:
+    const char* field_0;
+    al::LiveActor* mRootActor;
+    void* field_10;
```

Comment:
```suggestion
    bool field_10;
```
Something about if an action is running, I think - haven't looked to deeply though.

Reply:
Realized that `mRootActor` and `field_0` were also swapped here, fixed that. Named `field_10` `mIsActionRunning` based on what I saw.